src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
```

Comment:
Here some examples how this could be refactored:

```js
function calculateRentalCost(days) {
  const discount = days >= 7 ? 50 : days >= 3 ? 20 : 0;

  return days * 40 - discount;
}
```

or

```js
function calculateRentalCost(days) {
  let discount = 0;

  if (days >= 7) {
    discount = 50
  } else if (days >= 3) {
    discount = 20
  };

  return days * 40 - discount;
}
```

or as an arrow function

```js
const calculateRentalCost = (days) => days * 40 - (days >= 7 ? 50 : days >= 3 ? 20 : 0)
```